### ISSUE TITLE
Awful anti-aliasing on 125% scale

### ISSUE THREAD

**Issue by aaronliu0130** [developersu/ns-usbloader#153]
**Describe the bug (mandatory)**
The text is very thin and perhaps not anti-aliased, making it very hard to read

**Information about environment (mandatory):**
 - OS: Windows 11 x64 with 125% scaling
 - Java version Temurin-20.0.2+9 (I symlinked the JDK directory while trying to fix it; this also happens for the bundled JRE)

**Steps to reproduce**
1. Set scaling to 125%
2. Open app

**Expected behavior**
Normal text

**Screenshots and/or information from application log**
![image](https://github.com/developersu/ns-usbloader/assets/11874211/69530046-aaf3-4b9a-819a-601c9f149409)

**Did you see this issue in previous versions?**
Don't know

**Additional context**
I've also tried changing the executable's high DPI settings.

**Comment by developersu** [comment]
@aaronliu0130 how did you perform scaling? Some Windows-specific way or via options like `-Dsun.java2d.uiScale.enabled=true  -Dsun.java2d.uiScale=1.5`?

**Comment by aaronliu0130** [comment]
Windows.
![image](https://github.com/developersu/ns-usbloader/assets/11874211/1ea3160c-30f7-4aa8-b6e4-1904eaf67b42)

**Comment by developersu** [comment]
Try [this](https://superuser.com/questions/988379/how-do-i-run-java-apps-upscaled-on-a-high-dpi-display/1207925#1207925) or if it still looks weird you can execute jar from cmd.exe with options noted above.
E.g. `java like -Dsun.java2d.uiScale.enabled=true -Dsun.java2d.uiScale=1.5 -jar ..\Downloads\APPLICATION_NAME.jar` where `..\Downloads\APPLICATION_NAME.jar` path to jar you downloaded.

I thought in latest openJFX HiDPI supported well.. If it won't help let me know, I'll make build with the most recent libraries. Maybe that's the problem. 
These fonts are system-default picked by java automatically.

**Comment by aaronliu0130** [comment]
The link doesn't work, and I don't have a jar on Windows. After downloading and opening it (with no options) it tells me that `Error. A JNI error has occurred, please check your installation and try again`

**Comment by developersu** [comment]
There is usually exception stack trace following this message. Have you seen any?

**Comment by aaronliu0130** [comment]
Nope. After closing that pop up I just got some generic JVM error. I’ll copy down the exact error message when I get home

**Comment by developersu** [comment]
Try running it via cmd.com or powershell terminal.. there are should be more info

**Comment by aaronliu0130** [comment]
Ah, it seems like I had multiple version of JDK installed and Windows automatically launches with the last one updated, which was 8. I've reinstalled the latest JDK and now it launches.
The example command doesn't work though. In cmd it's like nothing changed and I've tried both 1.25 and 1.5 for the UI scale argument (also assuming "like" isn't part of the command). In powershell it didn't even recognize the argument. It also always gave the following warning:
`WARNING: Unsupported JavaFX configuration: classes were loaded from 'unnamed module @7cca494b'`
I also have Rectify11 installed, which changes DPI behavior for older win32 apps like the control panel. I doubt it applies to Java though.

**Comment by aaronliu0130** [comment]
![image](https://github.com/developersu/ns-usbloader/assets/11874211/00d9722e-6bab-4d65-bc6f-c40ccbd0c52e)
There seems to be a difference between default and system and system (enhanced) scaling: the latter two are blurry

**Comment by developersu** [comment]
Ok, try using another font by selecting it in settings. Grab latest builds [from here](https://redrise.ru/builds/ns-usbloader/). 'jar' or 'Installer' with "-7.1-" in name (or whatever with latest timestamp).

**Comment by aaronliu0130** [comment]
Great, that worked! Some fonts on my computer like San Francisco didn't show up in the selector's list though...

**Comment by developersu** [comment]
¯\_(ツ)_/¯
